WPThemeReview/Sniffs/PluginTerritory/ForbiddenFunctionsSniff.php

```diff
@@ -16,6 +16,7 @@
  *
  * @link  https://make.wordpress.org/themes/handbook/review/required/#presentation-vs-functionality
  *
+ * @since 0.2.0 Updated the list of forbidden functions.
```

Comment:
WP Docs convention is to have these tags in chronological order, not in reverse order.